string[3] name
float64[3] position
float64[3] velocity
float64[3] acceleration 
